src/styles/Home.css

```diff
@@ -147,3 +147,50 @@
 		max-height: 8rem;
 	}
 }
+
+/* ========== CurrEvents.jsx ========== */
+.section-title {
+	font-weight: bold;
+	margin: 2rem 0;
+	font-size: 2.5rem;
+	margin-bottom: 0.2rem;
+}
+
+.events-container {
+	display: flex;
+	flex-wrap: wrap;
+	gap: 1rem;
+	margin-bottom: 2rem;
+}
+
+.event-column {
+	flex: 1 1 45%;
+	padding: 1rem;
+	border-radius: 8px;
+}
+
```

Comment:
Let's add a CSS selector here for `.event-img` (which you've already defined in CurrEvents.jsx). We should make sure that the two images are the same size.